### PR TITLE
Desktop homepage

### PR DESCRIPTION
**Pull request by Mike-devloper**
Add media query to styles.css file
Update index.html file and image files for desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -185,103 +190,96 @@ <h3 class="work_title">Multi-Post Stories Gain+Glory</h3>
...
+                <div >
+                    <div class="boxed_flex">
+                    <div class="lang_boxed">Bootstrap</div>
+                    <div class="lang_boxed">Ruby on rails</div>
+                    <div class="lang_boxed">RSpec</div>
+                    <div class="lang_boxed">Capybara</div>
+                    <div class="lang_boxed">Selenium</div>
+                    </div>
+                </div>           
             </div>
```
Comment (roshan-bajgain) [inline index.html:238]: - [ ] I would suggest please decrease the gap between the list and provide padding on it. See image below for reference.
### Your design
your
### Figma design
figma

File: index.html [hunk 2]
```diff
@@ -185,103 +190,96 @@ <h3 class="work_title">Multi-Post Stories Gain+Glory</h3>
...
+                 <div class="center1" >
+                    <form action="https://formspree.io/f/xvoyggzp" method="POST" id="form">
+                        <input type="text" name="name" placeholder="full name" maxlength="30" class="stretch" required>
+                        <input type="email" name="email" placeholder="Email address"  class="stretch" required>
+                        
+                        <textarea name="message" maxlength="500" placeholder="Enter text here" class="comment stretch" required></textarea>
+                        
+                        <div class="button_left">
+                            <button type="submit" class="button3 email_flex">Get in touch</button>
+                        </div>
```
Comment (roshan-bajgain) [inline index.html:274]: - [ ] I would suggest you please decrease the spacing between the `Get in touch` buttons and the input field and decrease the padding left and right as well.
### Image
hiiiiiiiiisssssss

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Mike-devloper, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
- [ ] I would suggest you please make changes to whatever I gave on the desktop version as it is creating the same problems on the mobile version as well. Please fix it
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Mike-devloper , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used

## Optional suggestions

- [ ] [optional] @Mike-devloper great work implementing the changes requested, kindly check your readme file, and fix the title and description section.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (oguntayomicheal) [reply index.html:274]: Hi @roshan-bajgain, thanks for the review, all suggested changes will be implemented.

**Comment by oguntayomicheal** [comment]
Hi @danmainah, thanks for approving my code. I will implement the optional changes.

**Pull request merged**